From d5c460c31f4806238b38f4466cfbe143d07b1585 Mon Sep 17 00:00:00 2001
From: Robin Gong <yibin.gong@nxp.com>
Date: Thu, 5 Jan 2017 14:09:19 +0800
Subject: [PATCH 1382/5242] MLK-13720-3: regulator: pf1550-regulator-rpmsg:
 use the specific pf1550 rpmsg channel name

commit  5835c15e74d6f396ad688758eb35098bbc073701 from
https://source.codeaurora.org/external/imx/linux-imx.git

Since multi rpmsg instances supported now, we can use the specific rpmsg
channel name "rpmsg-regulator-channel" instead of "rpmsg-openamp-demo-channel"

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
(cherry picked from commit 8a9d73fe43da2833951b428f389aac01aef7cb2a)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/regulator/pf1550-regulator-rpmsg.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/regulator/pf1550-regulator-rpmsg.c b/drivers/regulator/pf1550-regulator-rpmsg.c
index a26442e..fd0f69d 100644
--- a/drivers/regulator/pf1550-regulator-rpmsg.c
+++ b/drivers/regulator/pf1550-regulator-rpmsg.c
@@ -326,8 +326,7 @@ static void rpmsg_regulator_remove(struct rpmsg_device *rpdev)
 }
 
 static struct rpmsg_device_id rpmsg_regulator_id_table[] = {
-	/* { .name	= "rpmsg-regulator-channel" }, */
-	{ .name	= "rpmsg-openamp-demo-channel" },
+	{ .name	= "rpmsg-regulator-channel" },
 	{ },
 };
 
-- 
1.7.9.5

